table/two_level_iterator.cc

```diff
@@ -154,14 +154,8 @@ void TwoLevelIterator::Prev() {
 
 
 void TwoLevelIterator::SkipEmptyDataBlocksForward() {
-  while (second_level_iter_.iter() == nullptr ||
-         (!second_level_iter_.Valid() &&
-         !second_level_iter_.status().IsIncomplete())) {
-    // Move to next block
-    if (!first_level_iter_.Valid()) {
```

Comment:
Can you explain why you can assume that `first_level_iter_.Valid()`?

Preferably in the comments in the code.